vendor/opam-file-format/src/dune

```diff
@@ -1,6 +1,20 @@
 (library
- (name opam_file_format)
- (wrapped false))
+  (name opam_file_format)
```

Comment:
Should be single space

Reply:
I didn't want to reformat the dune file in our [upstream fork](https://github.com/ocaml-dune/opam-file-format) for ease of rebasing onto newer releases, shall I reformat it anyway?